### PR TITLE
Pre-demo changes

### PR DESCRIPTION
**Pull request by slimslenderslacks**
Changes added for the demo.

### CODE REVIEW

File: pdf_bot.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
 
 embeddings, dimension = load_embedding_model(
-    embedding_model_name, config={"ollama_base_url": ollama_base_url}, logger=logger
+    embedding_model_name, config={ollama_base_url: ollama_base_url}, logger=logger
```
Comment (oskarhane) [inline pdf_bot.py:34]: I don't think we want this change.

File: chains.py [hunk 2]
```diff
@@ -59,10 +58,10 @@ def configure_llm_only_chain(llm):
     # LLM only response
     template = """
     You are a helpful assistant that helps a support agent with answering programming questions.
-    If you don't know the answer, just say that you don't know, don't try to make up an answer.
+    If you don't know the answer, just say that you don't know, you must not make up an answer.
     """
     system_message_prompt = SystemMessagePromptTemplate.from_template(template)
-    human_template = "{text}"
+    human_template = "{question}"
```
Comment (oskarhane) [inline chains.py:64]: Is the `{question}` keyword implicitly replaced with the first argument in `chain.invoke()`?

File: chains.py [hunk 3]
```diff
@@ -47,6 +45,7 @@ def load_llm(llm_name: str, logger=BaseLogger(), config={}):
             base_url=config["ollama_base_url"],
             model=llm_name,
             streaming=True,
+            seed=2,
```
Comment (jexp) [inline chains.py:48]: we can remove this one or comment it out.

**Review by jexp — APPROVED** [review]
@nfcampos can you have a look at the {text} -> {question} change?

File: chains.py (hunk 2 above)
Comment (slimslenderslacks) [reply chains.py:64]: I think this is right but it'd be good to get confirmation from @nfcampos - @jmorganca @mchiang0610 - this is one of the changes that was on the mac studio.  Do you know who should review this?

**Pull request merged**